src/framework/mocha/Utilities/MOUtilities.m

```diff
@@ -980,7 +980,7 @@ id MOGetBlockForJavaScriptFunction(MOJavaScriptObject *function, NSUInteger *arg
             return nil;
         }
         
-        JSValueRef *jsArguments = (JSValueRef *)malloc(sizeof(JSValueRef) * (functionArgCount - 1));
+        JSValueRef *jsArguments = (JSValueRef *)malloc(sizeof(JSValueRef) * (functionArgCount));
```

Comment:
This didn't necessarily affect this specific situation, but allocating space for one fewer than the number of parameters was surely not intended